cabal.project

```diff
@@ -39,3 +39,7 @@ benchmarks: true
 program-options
   ghc-options: -Werror
 
+-- Windows cross with bitvec/simd is broken.
+package bitvec
+  -- Workaround for windows cross-compilation
+  flags: -simd
```

Comment:
Do you know if this is a valid syntax for `cabal.project`? Never tried it before, but if it works it would be much better to turn it off only on platforms that are problematic. The concern I have is that if we do use `simd` on linux and mac for the actual `cardano-node` then all of the tests in this repo are not tested with the same flags
```suggestion
  if os(windows)
    flags: -simd
```

Reply:
This sadly does not work.